app/helpers/mb_form_builder.rb

```diff
@@ -20,6 +20,24 @@ def mb_input_field(
     HTML
   end
 
+  def mb_textarea(
+    method,
+    label_text,
+    notes: [],
+    options: {},
+    classes: [],
+    placeholder: nil,
+    autofocus: nil
+  )
+    classes = classes.append(%w[textarea])
+    <<-HTML.html_safe
+      <fieldset class="form-group#{error_state(object, method)}">
+        #{label_and_field(method, label_text, text_area(method, { autofocus: autofocus, class: classes.join(' '), autocomplete: 'off', autocorrect: 'off', autocapitalize: 'off', spellcheck: 'false', placeholder: placeholder }.merge(options)), notes: notes)}
```

Comment:
yeah you are right, weird that it didnt correct. Copied this over from GCF.

Anyway, cleaned up at 5371e9e